### PR TITLE
Fixed the repo

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-ast
```
Comment (Cadair) [inline .pre-commit-config.yaml:5]: There is no ast
```suggestion
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      - id: check-case-conflict
+      - id: trailing-whitespace
+      - id: check-yaml
+      - id: debug-statements
```
Comment (Cadair) [inline .pre-commit-config.yaml:9]: Or debug statements
```suggestion
```

File: SEP-0000.md [hunk 3]
```diff
@@ -1,29 +0,0 @@
-# SEP-0000 - Index of SunPy Enhancement Proposals (SEPs)
```
Comment (Cadair) [inline SEP-0000.md:1]: Has this been deleted because it's also in the readme?

**Review by Cadair — COMMENTED** [review]
Looks good to me, I would like input from someone who knows about Zenodo to know if we should re-publish them with these updates or if it doesn't matter if the repo version and the zenodo version diverge in minor ways.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (nabobalis) [reply .pre-commit-config.yaml:5]: This is why I get for copying the sunpy one.

File: SEP-0005.md [hunk 4]
```diff
@@ -3,14 +3,15 @@
 | SEP           | 5                                                                     |
 |---------------|-----------------------------------------------------------------------|
 | title         | Coordinates Module                                                    |
-| author(s)     | Pritish Chakraborty<br>[Stuart Mumford](https://orcid.org/0000-0003-4217-4642)                                   |
+| author(s)     | Pritish Chakraborty <br> [Stuart Mumford](https://orcid.org/0000-0003-4217-4642)                                   |
 | contact email | chakrabortypritish@gmail.com, stuart@mumford.me.uk                    |
 | date-creation | 2014-05-18                                                            |
 | type          | standard                                                              |
-| discussion    | https://groups.google.com/forum/m/#!topic/sunpy-gsoc-2014/hKWRYe1ND00 |
+| discussion    | <https://groups.google.com/forum/m/#!topic/sunpy-gsoc-2014/hKWRYe1ND00> |
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply SEP-0005.md:10]: Good question, I think I was planning to do something with this but forgot.

**Comment by nabobalis** [comment]
It has a copy of the MD file and a md5 hash for it: https://zenodo.org/record/7020103 

So I don't think any republishing or updating is needed since there has been no material change to these files.

File: LICENSE.md [hunk 5]
```diff
@@ -5,24 +5,23 @@ SunPy developers and is distributed subject to the terms of the Creative Commons
 Attribution 4.0 International Public License which is reproduced
```
[the remark this answers is not in the document]
Comment (Cadair) [reply LICENSE.md:5]: I copied it from a place that has markdown versions of CC licenses.

[the remark this answers is not in the document]
Comment (nabobalis) [reply LICENSE.md:5]: They clearly need to markdown correctly.

**Pull request merged**